start.html

```diff
@@ -178,145 +181,341 @@ <h3 class="accordion">
+                         installed already. If you are not sure, we
+                         recommend that you run this step after saving
+                         any existing repo files and compare the new
+                         ones with the existing ones. This should be a
+                         one-time procedure.
+                       </p>
+                       <p class="info_text"  style="text-align:justify">
+                         <span class="c_snip_inline">pbench-agent</span> has a large number
+                         of dependencies. The first time that it is
+                         installed, it will install a long list of
+                         packages that it depends on: it is not
+                         unusual for the list to contain close to two
+                         hundred packages.  So before you install
+                         <span class="c_snip_inline">pbench-agent</span>,
+                         you need to ensure that the host contains a
+                         reasonable set of repos
+                         in <span class="c_snip_inline">/etc/yum.repos.d</span>.
+                         For example, for RHEL8, you will need the CRB
+                         (Code Ready Builder) repo; for CentOS 8, you
+                         will need the Centos-PowerTools repo.
+                       </p>
+                       <p class="info_text"  style="text-align:justify">
+                         In general, you will need some Perl packages and a lot
+                         of Python3 packages, in addition to some basic packages
+                         that the pbench-agent spec file lists.
+                       </p>
+                     </div>
+                   </div>
+<!-- Section: Installing using ansible  -->
+                   <div class="accordionCard">
+		     <h3 class="accordion">
+		       <button class="accordionBtn pf-c-accordion__toggle" aria-expanded="false">
+			 <i class="arrowIcon fas fa-angle-right pf-c-accordion__toggle-icon" aria-hidden="true"></i>
+			 <span class="pf-c-accordion__toggle-text">Installing using ansible</span>
+		       </button>
+		     </h3>
+		     <div class="explanation pf-c-accordion__expanded-content pf-m-fixed" hidden>
+                       <p class="info_text"  style="text-align:justify">
+                         Once you have a good set of repos installed on your machines, then you can
+                         install <span class="c_snip_inline">pbench-agent</span>. The recommended method for installation is through
+                         ansible playbooks.
+                       </p>
+                       <p class="info_text"  style="text-align:justify">
+                         The steps are:
+                       </p>
+                       <div class="info_text" style="text-align:justify">
+                           <li>Make sure that you have ansible and ansible-galaxy installed on your
```

Comment:
Perhaps use monospace font for RPM references to `ansible` and `ansible-galaxy`?

Reply:
Fixed.